### PR TITLE
Update example

### PR DESCRIPTION
**Pull request by SoyGema**
- [X] ❗ I have followed the [Contributing to DVCLive](https://github.com/iterative/dvclive/blob/main/CONTRIBUTING.rst)
  guide.

- [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

### Notes

 Aims to solve #663 

- Add sections and key bullet points about what the example does.

- Following some of the dynamics from[ Hugginface notebooks](https://github.com/huggingface/notebooks) examples 
 i̵n̵ ̵w̵h̵i̵c̵h̵ ̵c̵o̵n̵f̵i̵g̵u̵r̵a̵t̵i̵o̵n̵ ̵t̵r̵a̵c̵e̵ ̵h̵a̵s̵ ̵b̵e̵e̵n̵ ̵e̵r̵a̵s̵e̵d̵ ̵b̵u̵t̵ ̵M̵L̵ ̵r̵e̵s̵u̵l̵t̵s̵ ̵a̵r̵e̵ ̵s̵h̵o̵w̵n̵.̵ ̵

EDIT : Please see `codespell` limitations in notebooks scenario [#2138](https://github.com/codespell-project/codespell/issues/2138) that nudged towards submitting the notebook without being executed.
Please consider a separate repository with examples or studying the addition of tools like [reviewnb.com ](https://www.reviewnb.com/) in repositories that might contain notebooks and might be key for adoption.

Thanks for the time dedicated to reviewing the PR! 🙏

### CODE REVIEW

**Comment by SoyGema** [comment]
-  [ ] Test core not pass : related to studio ?
-  [ ] code spell : prob missspelling

**Comment by SoyGema** [comment]
Hello there.
Ultimately, I´ve submitted the Notebook without being executed. 
With that in mind, would like to encourage you to think about how to , at some point, solve this challenge (executed notebooks) in order to make tool adoption easier for data scientists.

**Comment by daavoo** [comment]
>  Test core not pass : related to studio ?

Test failures are unrelated to your changes. I am working on a fix

File: examples/DVCLive-HuggingFace.ipynb [hunk 1]
```diff
@@ -1,167 +1,321 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "CLRgy2W4dmsU"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install dvc pandas\n",
+        "!pip install accelerate datasets dvclive evaluate 'transformers[torch]' --upgrade"
```
Comment (daavoo) [inline examples/DVCLive-HuggingFace.ipynb:41]: DVC should be installed by dvclive so no need to specify it.
Merged pandas in the second line.

```suggestion
        "!pip install accelerate datasets dvclive evaluate pandas 'transformers[torch]' --upgrade"
```

File: examples/DVCLive-HuggingFace.ipynb [hunk 2]
```diff
@@ -1,167 +1,321 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "l29wqAaDdmsW"
+      },
+      "source": [
+        "### Comparing Experiments\n",
+        "\n",
+        "We create a dataset with the experiments in order to visualize it.\n"
```
Comment (daavoo) [inline examples/DVCLive-HuggingFace.ipynb:246]: Did you mean pandas dataframe?

```suggestion
        "We create a dataframe with the experiments in order to visualize it.\n"
```

**Review by daavoo — APPROVED** [review]
Thanks, Gema! LGTM. Left 2 minor comments

**Pull request merged**